File: py/1216.py
from sys import stdin, stdout


def main() -> None:
    input = stdin.readline
    print = stdout.write
    age_sum = 0
    count = 0
    while True:
        try:
            name = input()
            age = int(input())
        except ValueError:
            break
        age_sum += age
        count += 1
    age_sum *= 10
    count *= 10
    print(f'{(age_sum / count):.1f}\n')


if __name__ == '__main__':
    main()

File: cpp/1216.cpp
#include <iostream>
#include <stdlib.h>
#include <string>
#include <vector>

using namespace std;

int main()
{
    string name;
    int age, count = 0;
    long  long int age_sum = 0;
    while (getline(cin, name)) {
        cin >> age;
        getchar();
        age_sum += age;
        count++;
    }
    age_sum *= 10;
    count *= 10;
    printf("%.1f\n", (double)age_sum / count);
    return 0;
}